t/mws_range/mws_range.xml

```diff
@@ -0,0 +1,47 @@
+<math xmlns="http://www.w3.org/1998/Math/MathML" alttext="\range{1}{5}\range{}{5}\range{1}{}\range{}{}\range{1}{5}+\qvar@construct{x}%&#10;\range{9.00}{11.00}\range{47.50}{52.50}\range{-1.00}{1.00}" display="inline" xml:id="p1.1.m1.1">
+  <semantics xml:id="p1.1.m1.1a">
+    <mrow xml:id="p1.1.m1.1.11" xref="p1.1.m1.1.11.cmml">
+      <mrow xml:id="p1.1.m1.1.11.1" xref="p1.1.m1.1.11.1.cmml">
+        <mrow xml:id="p1.1.m1.1.1" xref="p1.1.m1.1.1.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.1a" xref="p1.1.m1.1.1.cmml">[</mo>1...5<mo mathcolor="red" xml:id="p1.1.m1.1.1b" xref="p1.1.m1.1.1.cmml">]</mo></mrow>
```

Comment:
Hm, the range snippets such as `1...5` ought to be in a `<mn>` element in the presentation, or some other, but definitely shouldn't free-float in an mrow